### DISCUSSION TITLE
Qiqqa website is down, I need urgent support

### DISCUSSION THREAD

**Discussion by arunendramishra** [jimmejardine/qiqqa-open-source#359] [Q&A]
Hi There, 

@GerHobbelt @jimmejardine I am in deep trouble. All my research work is saved on Qiqqa web and I am unable to access since Qiqqa website is down weeks. I need your support either extracting or sync my web library. Constant crashes are also an issue. Unable to find any email id of support contact. Please help.

Regards
Arun

**Comment by GerHobbelt** [comment]
See also #358.
